acceptance_tests/features/find_reporting_unit.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Find Reporting Unit
+   As an internal user
+   I need to be able to find a Reporting Unit
+   So that I can easily locate the RU to deal with issues
```

Comment:
I think we should be consistent with saying 'RU' or 'Reporting unit' In general I'm leaning towards the latter